docs/development/Undo-RecordObject-vs-EditorUtility-SetDirty.md

```diff
@@ -0,0 +1,7 @@
+# Which one should you use Undo-RecordObject or EditorUtility.SetDirty 
+
```

Comment:
this needs to be more information. I'm sure most don't even know what you are talking about, it lacks context.